### ISSUE TITLE
nircmd

### ISSUE THREAD

**Issue by romjacket** [tonywoode/emu-PC-switch-sync-tool#2]
I think nirsoft tools are the way to go for what you're trying to accomplish, but using soundvolumeview you do something like `for /f "tokens=1,2 delims=," %a in ('soundvolumeview.exe /scomma ""') do if "%~b"=="Device" do ((((__stuff__))))`
Can't wait to see where take this project.  I'm writing [this tool](https://romjacket.github.io/RJ_LinkRunner/index.html) which aims in parallel with what you've got going here.

**Comment by tonywoode** [comment]
Thanks for this - that's a great method for getting the sound device names - how do you think I should be using that - in my Profile Switcher you mean?

And your tool looks great: would you consider adding to [the readme](https://github.com/romjacket/RJ_LinkRunner) with an explanation of what it does (I couldn't quite grok from the auto-generated docs in what situation I would use it, and what the features are, and my AutoHK is pretty rusty..)

**Comment by romjacket** [comment]
yes.  using setsounddevice and a set /p or choice prompt you can simplify the sound-device selection during setup and use it otherwise the same way you currently use nircmd.
I'll do what i can to clarify my readme, but you may find the tool useful where you have a game (or group of games) which require unique pc settings (pre-run/post-run).
I'm trying to create a tool that will (as yours does) switch the monitor/audio-output/keymapper etc. but then also automatically retain any changes.  Ultimately I would also like it to automate cloud-syncing saves/game-settings.
